src/app/static/src/app/hintVersion.ts

```diff
@@ -1 +1 @@
-export const currentHintVersion = "1.56.2";
+export const currentHintVersion = "1.57.0";
```

Comment:
I don't think we actually need to update the version where there is a non-behavioural change like this one, but it doesn't really matter.